CMSgov/mdct-mcr
# MDCT-1891: Add dynamic entities to DrawerSections

## Description

Fixes [MDCT-1891](https://qmacbis.atlassian.net/browse/MDCT-1891)

A picture is worth 1000 words, so heres my million word essay

https://user-images.githubusercontent.com/19439679/190922859-ff43d8c1-3936-4b8a-8fd7-5949d1af41f9.mov

### How to test
./dev local
Create a program
Go to Section D and see that because theres no plans created you'll get a "No Plans Available" message.
Go to back to Section A and add as many plans as you want
Go back to Section D and see all your new plans in their dynamic glory.

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
